bigbite/Jake-Week-2
# Feat/additional features

## Description
Added additional features (such as modals on index page, dark mode theme) and resolved some issues (such as the footer on contact-page being positioned using margin instead of position absolute, alongside adding a box shadow variable.)


## Change Log



- Added Modals to each 'Learn More' button on index.html
- Added a dark theme toggled using a button on the footer.
- Resolved box-shadow repetition by adding a variable to be used globally.
- Resolved contact page footer being positioned using margins and instead used position absolute.

## Steps to test


Dark Mode:
The dark mode works by adding an '.is-dark-mode' class to the body of each page. This stores a key within local storage to ensure that when the page is closed, the theme preference is saved.

Modals:
The modals on Index.html work by checking which button is being pressed via a classList.contains(). It then opens the corresponding modal alongside a blurred background.  The close buttons for each modal work in the same way, they check for a class to define which modal close button is being pressed and close the modal accordingly.
## Screenshots/Videos
Dark Mode:

https://github.com/bigbite/Jake-Week-2/assets/53060167/946ccd94-37a8-4f0d-9208-796cf6286996

Modals:

https://github.com/bigbite/Jake-Week-2/assets/53060167/4e2998df-4ca1-464d-95ee-132b990a0c79




## Checklist:
